src/shogun/machine/Machine.cpp

```diff
@@ -26,6 +26,7 @@ CMachine::CMachine()
 	    &m_store_model_features, "store_model_features",
 	    "Should feature data of model be stored after training?");
 	SG_ADD(&m_data_locked, "data_locked", "Indicates whether data is locked");
+	SG_ADD(&m_supports, "m_supports", "Bitset representing machine properties");
```

Comment:
This is a property that shouldn’t be serialised as it remains unchanged during the instance life. So we can have a property for that maybe?
@gf712

Reply:
yes, that should be easy to add, just do it here:
https://github.com/shogun-toolbox/shogun/blob/6434fe2df14bb0dbc34c27b2da4c7147f67b1159/src/shogun/base/AnyParameter.h#L34-L42
and add the property in SG_ADD, i.e. `SG_ADD(..., ParameterProperties::NOT_SERIALISABLE)`

Reply:
Actually the best thing would be to just not register it I now realised. No get/set/clone/equals/save needed